src/components/CompareResults/beta/RunValues.tsx

```diff
@@ -23,34 +23,36 @@ const styles = {
 };
 
 function RunValues(props: RunValuesProps) {
-  const { name, mean, runValues: values } = props.revisionRuns;
+  const { name, median, values, stddev, stddevPercent } = props.revisionRuns;
   return (
     <div className={styles.container}>
       <div>
-        <b>{name}:</b> {mean}ms
+        <b>{name}:</b> {median} ms
       </div>
       <div>
         <GraphDistribution name={name} values={values} />
       </div>
       <div>
         <div className={styles.values}>
-          {values.map((v, index) => (
-            <div key={`${v}-${index}`} className={styles.value}>
-              {v}
+          {values.map((value, index) => (
+            <div key={`${value}-${index}`} className={styles.value}>
```

Comment:
I believe it's not really useful to use the value if we already use the index that changes for every line. Just use the index as the key.